heron/proto/physical_plan.proto

```diff
@@ -11,7 +11,12 @@ import "topology.proto";
 message StMgr {
   required string id = 1;
   required string host_name = 2;
+  // This is the data port for communication
+  // accross the containers(i.e. peer stmgrs)
   required int32  data_port = 3;
```

Comment:
stmgr_port and instance_port should be better. There are both data and control infos passed to and from these two ports.

Reply:
I prefer not to change any 'names' in this pr because that will pollute this pr. I will send out a follow up pr to fix the names

Reply:
👍